__tests__/app.test.js

```diff
@@ -368,3 +368,20 @@ describe('/api/comments/:comment_id', () => {
       });
   });
 });
+
+describe('/api/users', () => {
+  test('GET: 200 - Responds with an array of users with username, name, avatar_url properties', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body: { users } }) => {
```

Comment:
There is an assertion that needs to be here 👀 If the response body.users is an empty array, the assertions in the .forEach won't run and the test will automatically pass 🫨 So we need an assertion to check that the array has a length property.